Code/Source/svFSI/ComMod.h

```diff
@@ -1466,6 +1474,11 @@ class ComMod {
     /// @brief Stop_trigger file name
     std::string stopTrigName;
 
+    /// @brief Precomputed state-variable file name
+    std::string precompFileName;
+
+    /// @breif Precomputed state-variable field name
```

Comment:
Similarly here

Reply:
fixed spelling mistake in zasexton:precomputed-velocities-#202